troposphere/views/app.py

```diff
@@ -69,12 +69,22 @@ def _populate_template_params(request, maintenance_records, notice_t, disabled_l
         notice = notice_t[1] if not notice_t[2] else None
     logger.info("maintenance notice tuple: {0}".format(notice_t))
 
+    #NOTE: Now that we've moved this section from .js to Django, sentry configuration _could_ become more dynamic:
+    if settings.DEBUG:
+        sentry_dict = None
+    else:
+        sentry_dict = {
+            "dsn": "https://27643f06676048be96ad6df686c17da3@app.getsentry.com/73366",
+            "release":"205096b5fde3a47303ad8d1fef9ff8052cbbd7d4",
+        }
```

Comment:
@lenards thanks for pointing this out, thats handy information! As we start to scope out what "sentry the plugin" looks like, specifying the `key` and `project` might be better than the whole-form `dsn`